UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/BloodTypes/BloodType.cs

```diff
@@ -55,5 +65,30 @@ public float GetSpareGasCapacity(ReagentMix reagentMix, Reagent reagent = null)
 			}
 			return GetGasCapacity(reagentMix, reagent) - reagentMix[reagent];
 		}
+
+
+		public float CalculatePercentageBloodPresent(ReagentMix reagentMix)
+		{
+			var Percentage = (reagentMix[this] / reagentMix.Total);
```

Comment:
what if `this` is null in the `reagentMix` list? or is it impossible to be able to do that?

Reply:
it's impossible

Reply:
alright cool